{"name": "Вызов ошибки при передаче в функцию определения самого высокого супергероя некорректного типа данных (пол - bool, наличие работы - bool)", "status": "passed", "steps": [{"name": "Определить самого высокого супергероя в списке", "status": "broken", "statusDetails": {"message": "TypeError: Некорректный тип данных\n", "trace": "  File \"C:\\Users\\annes\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\allure_commons\\_allure.py\", line 202, in impl\n    return func(*a, **kw)\n  File \"C:\\Users\\annes\\PycharmProjects\\Ozon\\methods.py\", line 50, in find_tallest_superhero\n    raise TypeError(Messages.WRONG_TYPE_ERROR)\n"}, "parameters": [{"name": "gender", "value": "True"}, {"name": "is_employed", "value": "True"}], "start": 1753642167539, "stop": 1753642167539}], "parameters": [{"name": "gender", "value": "True"}, {"name": "work", "value": "True"}], "start": 1753642167539, "stop": 1753642167540, "uuid": "dfc4edbf-7b41-4198-bd80-dbe4e62074b2", "historyId": "b2a90306944d9a0a460f1cb1ef3f1f29", "testCaseId": "c20fec1fdd11c57bab09c9bd58c966f1", "fullName": "tests.TestTallestHero#test_tallest_hero_wrong_data_type", "labels": [{"name": "suite", "value": "tests"}, {"name": "subSuite", "value": "TestTallestHero"}, {"name": "host", "value": "Lenovo-IdeaPad"}, {"name": "thread", "value": "17504-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests"}], "titlePath": ["tests.py", "TestTallestHero"]}